collab-folder/src/view.rs

```diff
@@ -710,6 +714,18 @@ pub struct View {
   pub created_by: Option<i64>, // user id
   pub last_edited_time: i64,
   pub last_edited_by: Option<i64>, // user id
+  // this value used to store the extra data with JSON format
```

Comment:
you can use /// for the comment. they have different effect when you hover on the extra property